### ISSUE TITLE
Feature request: Export a select/particular sheet in a Dashboard

### ISSUE THREAD

**Issue by cjvegi** [tableau/server-client-python#1132]
**Help Requested**
Some of my views have multiple sheets. Does the functionality exist where I could export a select Sheet to CSV/excel from a given View? I could not find the resources either in the documentation or in the community. So, any help/guidance is helpful.

**Comment by anyoung-tableau** [comment]
The Tableau REST API doesn't allow for specifying the specific sheet when exporting data from a Dashboard view. It will always return the data from the first sheet on the dashboard. Is there any reason why you aren't exporting the data from your desired sheet directly instead of from the dashboard on which it exists?

**Comment by saadbahir** [comment]
Hey @anyoung-tableau
If i understand correctly, we can query all sheets within a workbook and then query the data from any specific sheet?

**Comment by jacalata** [comment]
@saadbahir yes - check out the example code in https://github.com/tableau/server-client-python/blob/master/samples/export.py

**Comment by jacalata** [comment]
I suspect that implementing this might require/be improved by implementing this request as well: https://github.com/tableau/server-client-python/issues/1045

**Comment by saadbahir** [comment]
@jacalata I check the code and it seems that the script exports the data of the first available sheet because the API doesn't expose the data of the other hidden sheet within a view

**Comment by zeyadalaa0** [comment]
Has there been any updates regarding this?, or could someone show any alternative method?
